client/app/vps/modal/snapshot/snapshot-take.html

```diff
@@ -1,22 +1,21 @@
-<cui-modal>
-    <form name="VpsSnapshotTakeForm" data-ng-submit="$ctrl.confirm()">
-        <cui-modal-header data-on-dismiss="$ctrl.cancel()"></cui-modal-header>
-        <cui-modal-body data-title=":: 'vps_configuration_snapshot_take_title' | translate">
-            <div class="text-center" data-ng-if="$ctrl.loader.save">
-                <oui-spinner></oui-spinner>
-            </div>
-            <div data-ng-if="!$ctrl.loader.save">
-                <cui-modal-text data-text="'vps_configuration_snapshot_take_step1_question' | translate: {serviceName: $ctrl.serviceName}"></cui-modal-text>
-                <label for="text" class="oui-label" data-translate="vps_configuration_snapshot_take_step1_description"></label>
-                <input type="text" class="oui-input" data-ng-model="$ctrl.snapshot.description">
+<form novalidate name="form" data-ng-submit="form.$valid && $ctrl.confirm()">
+    <oui-modal
+        data-title="{{:: 'vps_configuration_snapshot_take_title' | translate}}"
+        data-primary-label="{{ ::'common_confirm' | translate }}"
+        data-secondary-action="$ctrl.cancel()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.cancel()"
+        data-loading="$ctrl.save.loading">
 
-                <p class="oui-modal__body-text red" data-translate="vps_configuration_snapshot_take_step2_confirmation"></p>
-            </div>
-        </cui-modal-body>
-        <cui-modal-footer>
-            <cui-cancel-button data-text="'common_cancel' | translate" data-on-click="$ctrl.cancel()"></cui-cancel-button>
-            <cui-submit-button data-text="'common_confirm' | translate"
-                               data-disabled="$ctrl.loader.save"></cui-submit-button>
-        </cui-modal-footer>
-    </form>
-</cui-modal>
+        <p class="oui-modal__text" data-translate="vps_configuration_snapshot_take_step1_question"
+        data-translate-values="{serviceName: $ctrl.serviceName}"></p> 
+        <oui-field data-label="{{ ::'vps_configuration_snapshot_take_step1_description' | translate }}">
+            <input class="oui-input" 
+                type="text"
+                id="snapshotDescription"
+                name="snapshotDescription"
+                data-ng-model="$ctrl.model">
+        </oui-field>
+        <p class="oui-modal__text red" data-translate="vps_configuration_snapshot_take_step2_confirmation"></p>
```

Comment:
`.red`

Reply:
red is a class name, do we need . ??

Reply:
I meant we should remove it to use something semantically more relevant, like in the other managers 😃 . Maybe you could use this : https://getbootstrap.com/docs/4.1/utilities/colors/ if we have Bootstrap 4 utilities installed 😃

Reply:
A step further would be to remove a deprecated dependency: [`ovh-common-style`](https://github.com/ovh-ux/ovh-manager-cloud/blob/master/package.json#L88).